monkey/monkey_island/cc/setup/certificate/certificate_setup.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import os
+
+from common.utils.exceptions import InsecurePermissionsError
+from monkey_island.cc.services.utils.file_permissions import has_sufficient_permissions
+from monkey_island.cc.setup.island_config_options import IslandConfigOptions
+
+logger = logging.getLogger(__name__)
+
+
+def setup_certificate(config_options: IslandConfigOptions) -> (str, str):
```

Comment:
This function doesn't need the whole IslandConfigOptions object. It just needs 2 file paths.

We should decouple this from `IslandConfigOptions`. Forcing this function to accept an `IslandConfigOptions` object makes it less flexible. If it accepts 2 file paths, those two file paths could come from any source, not just the config.